source/index.html.md

```diff
@@ -237,6 +237,7 @@ error categories, the last two digits define specific errors.
 * 0372: Incorrect value type for field
 * 0373: Request has expired
 * 0374: Invalid 2FA recovery code
+* 0375: Invalid routing number
```

Comment:
This should be set to `reserved for internal use` or similar as it's not an error code our API users will ever see.

Reply:
Perfect, I've added a comment to mark it as reserved